README.md

```diff
@@ -98,8 +97,10 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
     * [CodePipeline](https://aws.amazon.com/codepipeline/) - 1 active pipeline per month
     * [DynamoDB](https://aws.amazon.com/dynamodb/) - 25GB NoSQL DB
     * [EC2](https://aws.amazon.com/ec2/) - 750 hours per month of t2.micro or t3.micro(12mo)
+    * [EC2](https://aws.amazon.com/ec2/faqs/#t4g-instances) - 750 hours per month of t4g.small until December 31, 2023
```

Comment:
we can drop this one, i dont really want to add time banded things here and then forget about them

Reply:
Happy to drop it, though I would leave it as I suspect Amazon will end up including it into the Free Tier, as shown by their successive expansions of it's validity period (take a look at [their initial blog post](https://aws.amazon.com/blogs/aws/new-t4g-instances-burstable-performance-powered-by-aws-graviton2/), which has since then ended up as a kind of permanent Free Trial).

Reply:
Best to remove it, there's a 1000 entries on the list I cannot really keep it in mind for ifs and maybes sorry

Reply:
Done